From: Simon McVittie <smcv@collabora.com>
Date: Thu, 5 Apr 2018 11:20:25 +0100
Subject: Spelling: fix spelling of specify

Detected by Debian's Lintian tool.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: GNOME/glib!354
Applied-upstream: 2.59.0, commit:7a903bd0445457f50cc7f4720d11516cb87a00c6
---
 gobject/gobject-query.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gobject/gobject-query.c b/gobject/gobject-query.c
index 2992567..fd45fba 100644
--- a/gobject/gobject-query.c
+++ b/gobject/gobject-query.c
@@ -99,7 +99,7 @@ static gint
 help (gchar *arg)
 {
   g_fprintf (stderr, "usage: gobject-query <qualifier> [-r <type>] [-{i|b} \"\"] [-s #] [-{h|x|y}]\n");
-  g_fprintf (stderr, "       -r       specifiy root type\n");
+  g_fprintf (stderr, "       -r       specify root type\n");
   g_fprintf (stderr, "       -n       don't descend type tree\n");
   g_fprintf (stderr, "       -h       guess what ;)\n");
   g_fprintf (stderr, "       -b       specify indent string\n");
